### DISCUSSION TITLE
make !important rule work on pseudo-classes

### DISCUSSION THREAD

**Discussion by aryankarim** [tailwindlabs/tailwindcss#8461] [Ideas]
right now this doesn't work:
`<input value="test" disabled class="!disabled:bg-red-400" />`

It would be nice having ! operator work for pseudo-classes too.

**Comment by RobinMalfait** [comment]
Hey! Thank you for your question!
Much appreciated! 🙏

The `!` always goes in front of the utility, not the variant. E.g.: `disabled:!bg-red-400`

**Reply by aryankarim** [reply]
Thank you! I should've scanned the docs more thoroughly. 😅 I just found it.
[https://tailwindcss.com/docs/configuration#important-modifier](https://tailwindcss.com/docs/configuration#important-modifier)
